config/application.rb

```diff
@@ -38,7 +38,5 @@ class Application < Rails::Application
 
     # Don't generate system test files.
     config.generators.system_tests = nil
-
-    config.active_support.cache_format_version = 7.1
```

Comment:
I assume it's safe to remove this because `load_defaults` is not set to `7.1`. Is that right? If so, it might be worth explaining in the commit note.

Reply:
Correct. I'm afraid it's too late to update the commit note now that it's in main.